build-system/tasks/runtime-test/runtime-test-base.js

```diff
@@ -173,6 +173,11 @@ function updateReporters(config) {
     config.reporters = ['mocha'];
   }
 
+  if (isCiBuild()) {
```

Comment:
This will happen on GH actions as well, which we don't want. You can use `isCircleciBuild()` instead.

Reply:
Great catch